lib/dune

```diff
@@ -0,0 +1,32 @@
+(* -*- tuareg -*- *)
+
+let coverage_rewriter ~full =
+  let is_coverage =
+    try
+      Jbuild_plugin.V1.run_and_read_lines "printenv BISECT_ENABLE" = ["YES"]
```

Comment:
This can use `Sys.getenv` instead of running an external command

Reply:
In `dune` files, we cannot use `#require` and therefore `Unix` any more.
The `dune` error suggested to me that I should use the `Jbuild_plugin.V1.run_and_read_lines` function instead.

Reply:
Thanks, fixed.